src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -39,7 +39,8 @@ object Boot extends App {
     val gcsDAO = new HttpGoogleCloudStorageDAO(
       gcsConfig.getString("secrets"),
       new FileDataStoreFactory(new File(gcsConfig.getString("dataStoreRoot"))),
-      gcsConfig.getString("redirectBaseURL")
+      gcsConfig.getString("redirectBaseURL"),
+      gcsConfig.getInt("deletedBucketCheckSeconds")
```

Comment:
The other option was to hardcode a number, which would also have garnered negative attention in PR ;)